src/java/org/apache/cassandra/schema/IndexMetadata.java

```diff
@@ -146,6 +150,13 @@ public void validate(TableMetadata table)
         }
     }
 
+    public String getIndexClassName()
```

Comment:
Now that we have this method, we could call it from `SecondaryIndexManger#createInstance(IndexMetadata)`, instead of calling to `IndexMetadata.expandAliases`.

Then we could inline `IndexMetadata.expandAliases` into `IndexMetadata#getIndexClassName`.